OpenCyphalPicoBase-firmware.ino

```diff
@@ -197,49 +212,51 @@ static uint16_t update_period_ms_analoginput0        =     500;
-const auto reg_rw_aux_update_period_ms_inputvoltage        = node_registry->expose("aux.update_period_ms.inputvoltage",        {}, update_period_ms_inputvoltage);
-const auto reg_rw_aux_update_period_ms_internaltemperature = node_registry->expose("aux.update_period_ms.internaltemperature", {}, update_period_ms_internaltemperature);
-const auto reg_rw_aux_update_period_ms_input0              = node_registry->expose("aux.update_period_ms.input0",              {}, update_period_ms_input0);
-const auto reg_rw_aux_update_period_ms_input1              = node_registry->expose("aux.update_period_ms.input1",              {}, update_period_ms_input1);
-const auto reg_rw_aux_update_period_ms_input2              = node_registry->expose("aux.update_period_ms.input2",              {}, update_period_ms_input2);
-const auto reg_rw_aux_update_period_ms_input3              = node_registry->expose("aux.update_period_ms.input3",              {}, update_period_ms_input3);
-const auto reg_rw_aux_update_period_ms_analoginput0        = node_registry->expose("aux.update_period_ms.analoginput0",        {}, update_period_ms_analoginput0);
-const auto reg_rw_aux_update_period_ms_analoginput1        = node_registry->expose("aux.update_period_ms.analoginput1",        {}, update_period_ms_analoginput1);
-const auto reg_rw_aux_update_period_ms_light               = node_registry->expose("aux.update_period_ms.light",               {}, update_period_ms_light);
+const auto reg_rw_cyphal_node_id                            = node_registry->expose("cyphal.node.id",                           {true}, node_id);
+const auto reg_rw_cyphal_node_description                   = node_registry->expose("cyphal.node.description",                  {true}, node_description);
+const auto reg_rw_cyphal_pub_inputvoltage_id                = node_registry->expose("cyphal.pub.inputvoltage.id",               {true}, port_id_input_voltage);
+const auto reg_ro_cyphal_pub_inputvoltage_type              = node_registry->route ("cyphal.pub.inputvoltage.type",             {true}, []() { return "cyphal.primitive.scalar.Real32.1.0"; });
+const auto reg_rw_cyphal_pub_internaltemperature_id         = node_registry->expose("cyphal.pub.internaltemperature.id",        {true}, port_id_internal_temperature);
+const auto reg_ro_cyphal_pub_internaltemperature_type       = node_registry->route ("cyphal.pub.internaltemperature.type",      {true}, []() { return "cyphal.primitive.scalar.Real32.1.0"; });
+const auto reg_rw_cyphal_pub_input0_id                      = node_registry->expose("cyphal.pub.input0.id",                     {true}, port_id_input0);
+const auto reg_ro_cyphal_pub_input0_type                    = node_registry->route ("cyphal.pub.input0.type",                   {true}, []() { return "cyphal.primitive.scalar.Bit.1.0"; });
+const auto reg_rw_cyphal_pub_input1_id                      = node_registry->expose("cyphal.pub.input1.id",                     {true}, port_id_input1);
```

Comment:
@pavel-kirienko please correct me, but the `{true}` is optional in case of a const lambda that always returns the same value?

Reply:
It is not optional because it informs the register service about the value of the `persistent` flag. If you remove it, external nodes will see your persistent registers as non-persistent.

Reply:
Perfect, thanks for the clarification.